department-of-veterans-affairs/abd-vro
# Address medium severity secrel issues

## What was the problem?
The dependencies were leading to some medium severity SecRel issues which will be disallowed by the SecRel team starting in late October.

Associated tickets or Slack threads:
- [#205](https://github.com/department-of-veterans-affairs/abd-vro-internal/issues/205)

## How does this fix it?[^1]

Addresses dependency versions

## How to test this PR
- Run SecRel and check for any medium vulnerabilities in Aqua


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
